Makefile

```diff
@@ -336,12 +343,24 @@ $(INSTALL_BIN)/%: $(IELE_BIN)/%
 $(INSTALL_LIB)/%: $(IELE_LIB)/%
 	install -D $< $@
 
-install: $(patsubst %, $(INSTALL_BIN)/%, $(install_bins)) $(patsubst %, $(INSTALL_LIB)/%, $(install_libs))
+$(INSTALL_BIN)/iele-interpreter: $(INSTALL_LIB)/standalone/iele-testing-kompiled/syntaxDefinition.kore
+
+$(INSTALL_BIN)/iele-check: $(INSTALL_LIB)/check/well-formedness-kompiled/compiled.bin
+$(INSTALL_BIN)/iele-check: $(INSTALL_LIB)/check/well-formedness-kompiled/interpreter
+
+$(INSTALL_BIN)/kiele: $(INSTALL_LIB)/kore-json.py
+$(INSTALL_BIN)/kiele: $(INSTALL_LIB)/version
```

Comment:
Did you have a chance to test that this works for:

-   When the well-formedness checker passes _and_ when it fails?
-   Same for the interpreter (exit code 0, no printing, exit code non-zero, printing configuration)?

No worries if not, we can make a follow-up PR to make all that work. Just wondering.

Reply:
The Nix test suite runs `test-interactive`, `test-iele`, `test-wellformed`, and `test-illformed`. I don't know which of your points that covers.